repobee_sanitizer/_sanitize_repo.py

```diff
@@ -136,6 +142,14 @@ def _sanitize_to_target_branch(
         )
 
 
+def _check_repo_state(repo_root) -> str:
```

Comment:
The type hint here is incorrect, because the function may None, which means that the hint should be `Optional[str]`.

Reply:
```suggestion
from typeing import Optional
def _check_repo_state(repo_root) -> Optional[str]:
```